nfsstat/tests/test_e2e.py

```diff
@@ -0,0 +1,16 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from .common import CONFIG, METRICS
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check):
+    aggregator = dd_agent_check(CONFIG)
+
+    for metric in METRICS:
+        aggregator.assert_metric(metric)
+
+    aggregator.assert_all_metrics_covered()
```

Comment:
Could you also add this ?

```python
            from datadog_checks.dev.utils import get_metadata_metrics
            aggregator.assert_metrics_using_metadata(get_metadata_metrics())
```

Reply:
Seems like `.assert_metrics_using_metadata()` is not the the API docs already? 🤔 Probably missing a docstring to appear there https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.stubs.aggregator.AggregatorStub

Reply:
@florimondmanca There is a doc string though https://github.com/DataDog/integrations-core/blob/a58781f3d495deb490d81a354bd92cd1cb80e8a5/datadog_checks_base/datadog_checks/base/stubs/aggregator.py#L308-L321